backend/src/main/java/gov/cdc/usds/simplereport/api/testresult/ResultDataResolver.java

```diff
@@ -12,4 +17,15 @@ public class ResultDataResolver {
   public ApiFacility getFacility(TestResultsListItem result) {
     return new ApiFacility(result.getFacility());
   }
+
+  @SchemaMapping(typeName = "Result", field = "surveyData")
+  public ResolvedSurveyData getSurveyData(TestResultsListItem result) {
+    return new ResolvedSurveyData(result.getSurveyData());
+  }
+
+  @SchemaMapping(typeName = "Result", field = "patientLink")
```

Comment:
Do we need the `patientLink` field? I don't think we are using it on the results page or mapping it to the CSV when we download the results 🤔

Reply:
I'm pretty sure there was some error it ran into with the tests, but I can't remember what it specifically was so maybe that error was made obsolete by other changes, gonna test out removing that patient link then. `patientLink` is on the `TestResult` type which might be on the path to being deprecated in favor of `Result` now

Reply:
For [TestResultTest](https://github.com/CDCgov/prime-simplereport/blob/ad1df86c0e8669c27025f812f9523441c3f6b11b/backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/TestResultTest.java#L81), this line expects `patientLink` to be there on the result. This old unit test might be left over from when we used to need the patient link on the result but maybe we no longer do? Do you think we should edit/remove the unit test or add patient link for now?

Reply:
Ahhhh oops sorry for missing that! Thank you for looking into that again 😓 I think maybe for now we can put the `patientLink` field back so the tests could pass and this bug fix can be merged in. Thank you for your patience with me on this 😅 🙏 

But if you think it'll be helpful, we can make a tech debt ticket to clean this up. That field doesn't seem to be used (based on my rough reading of the codebase) but probably best to dig more thoroughly.

Reply:
Sounds good! I'll create a new ticket to address some of that tech debt around `TestResult` vs `Result` overall